examples/phi2/README.md

```diff
@@ -2,8 +2,19 @@
 
 ## Get the model
 
+Install the model builder script dependencies
+
+```bash
+pip install numpy
+pip install transformers
+pip install torch
+pip install onnx
+pip install onnxruntime
+```
+
 ```bash
-python -m onnxruntime_genai.scripts.export.py -m microsoft/phi-2 -e cpu -p int4 -o model/model.onnx
+wget https://github.com/microsoft/onnxruntime-genai/blob/main/src/python/models/export.py
+python export.py -m microsoft/phi-2 -e cpu -p int4 -o model/model.onnx
```

Comment:
consider being explicit about which directory the user should be in. this directory is assumed with the relative path `model/model.onnx`. similar comment for running phi2.py. maybe just have initial instructions for the user to change to this directory.